src/workerd/api/sockets.h

```diff
@@ -103,7 +127,31 @@ class Socket: public jsg::Object {
   jsg::Ref<ReadableStream> readable;
   jsg::Ref<WritableStream> writable;
   kj::Promise<kj::Own<kj::AsyncIoStream>> processConnection();
```

Comment:
Nitpick: Can you move this function declaration to appear after all of the member variables, to keep them nicely grouped?